spec/services/exception_hunter/error_creator_spec.rb

```diff
@@ -15,14 +15,14 @@ module ExceptionHunter
                    message: 'Something went very wrong 567')
           end
 
-          it 'returns true' do
-            expect(subject).to be true
-          end
-
           it 'creates an error ' do
             expect { subject }.to change(Error, :count).by(1)
           end
 
+          it 'returns the error' do
+            expect(subject).to eq(Error.first)
```

Comment:
This might cause race conditions use:
```suggestion
            expect(subject).to be_an(Error)
```